client/src/Main.elm

```diff
@@ -1801,6 +1736,46 @@ level0Decoder =
         |> JDP.required "references" (Decode.list referenceDecoder)
 
 
+backboneDecoder : Decode.Decoder Backbone
+backboneDecoder =
+    let
+        decodeOverhang : String -> Decode.Decoder Bsa1Overhang
+        decodeOverhang str =
+            case stringToBsa1Overhang (String.trim str) of
+                Just oh ->
+                    Decode.succeed oh
+
+                _ ->
+                    Decode.fail "Not a valid overhang"
+
+        decodeBsmb1Overhang : String -> Decode.Decoder Bsmb1Overhang
+        decodeBsmb1Overhang bsmb1_str =
+            case stringToBsmb1Overhang (String.trim bsmb1_str) of
+                Just bsmb1 ->
+                    Decode.succeed bsmb1
+
+                _ ->
+                    Decode.fail "Not a valid Gateway site"
+    in
+    Decode.succeed Backbone
+        |> JDP.required "name" Decode.string
+        |> JDP.required "id" (Decode.int |> Decode.map String.fromInt)
```

Comment:
Huh? Surely this should be an `Int` then?

Reply:
To be honest, I copied this from the level0Decoder we made together. So I figured it would have been a string here as well...

Reply:
I changed it to an ` Int` but I had to make some changes in the model and views.
I've pushed a new commit.

Reply:
I missed it. It should indeed be `mpg_number `and not `id`.